### PR TITLE
replace ci and release pipeline

### PR DESCRIPTION
**Pull request by rob9315**
(hopefully) fixes binary releases and makes pushing an update a manual operation, not triggering on every push

### CODE REVIEW

**Review by themoonisacheese — APPROVED** [review]
If that is functional, sounds good to me.

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,128 @@
+---
+name: Release
+
+on:
+  workflow_dispatch:
+    inputs:
+      version-type:
+        description: How major the changes are [ major | minor | patch | prerelease ]
```
Comment (KozmikNano) [inline .github/workflows/release.yml:8]: So will we have to specify in our commit, answer a issue or it will just work?

Comment (rob9315) [reply .github/workflows/release.yml:8]: you can run it from the actions tab manually like this
image

Comment (KozmikNano) [reply .github/workflows/release.yml:8]: Ok cool!

**Review by KozmikNano — APPROVED** [review]
Well i am going to push and hope that it works! Thanks for the help!

**Comment by nickcat325** [comment]
Thank you Rob.

**Pull request merged**